chatterbot/adapters/storage/mongodb.py

```diff
@@ -173,11 +173,11 @@ def filter(self, **kwargs):
 
         return results
 
-    def update(self, statement):
+    def update(self, statement, force = False):
```

Comment:
Is this style consistent with rest of the code base?
I have seen foo=bar in method parameters at quite a few places in this code base. 
Maybe @gunthercox could take the call.